tests/ui/borrowck/issue-92157.stderr

```diff
@@ -1,5 +1,5 @@
 error[E0308]: lang item `start` function has wrong type
-  --> $DIR/issue-92157.rs:11:1
+  --> $DIR/issue-92157.rs:12:1
```

Comment:
Discussion [DIAGNOSTICS]: related, but this test is also using `no_core` but for diagnostics purposes I think. I suppose there's no diff here because there isn't logic that would emit different diagnostics for these traits/lang items depending on e.g. diagnostic attributes *yet*.

Reply:
Looking at the original issue, it's an ICE regression test, so I think it's fine for the diagnostic output to have the potential of changing here.

Reply:
Sure.